### PR TITLE
Add beta1 relnotes and update breaking changes

### PR DESCRIPTION
**Pull request by dedemorton**
Also made a small change to changelog. `_node/settings` returns an error. `_node/logging` works.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## 5.0.0-beta1 (Sep 21, 2016)
  - Migrated Logstash's internal logging framework to Log4j2. This enhancement provides the following features:
    - Support changing logging level dynamically at runtime through REST endpoints. New APIs have been exposed 
-     under `_node/settings` to update log levels. Also a new endpoint `_node/logging` was added to return all 
+     under `_node/logging` to update log levels. Also a new endpoint `_node/logging` was added to return all
```
Comment (suyograo) [inline CHANGELOG.md]: Actually this is `node/settings` not `logging`.

File: docs/static/breaking-changes.asciidoc [hunk 2]
```diff
@@ -12,6 +12,21 @@ after installing it via a package (RPM, DEB).
 respectively. Previously it used to install in `/opt` directory. This change was done to make the user experience 
 consistent with other Elastic products. Full directory layout is described in <<dir-layout>>.
 
+**Default Logging Level:** Changed the default log severity level to INFO instead of WARN to match Elasticsearch. Existing logs
+(in core and plugins) were too noisy at INFO level, so we had to audit log messages and switch some of them to DEBUG
+level.
+
+**Index Template:** The index template for 5.0 has been changed to reflect Elasticsearch's mapping changes. Most
```
Comment (suyograo) [inline docs/static/breaking-changes.asciidoc]: Can we link `Elasticsearch's mapping changes` to https://www.elastic.co/guide/en/elasticsearch/reference/5.0/breaking_50_mapping_changes.html#_literal_string_literal_fields_replaced_by_literal_text_literal_literal_keyword_literal_fields

File: docs/static/breaking-changes.asciidoc [hunk 3]
```diff
@@ -12,6 +12,21 @@ after installing it via a package (RPM, DEB).
...
+importantly, the subfield for string multi-fields has changed from `.raw` to `.keyword` to match Elasticsearch's default
+behavior. The impact of this change to various user groups is detailed below:
+
+* New Logstash 5.0 and Elasticsearch 5.0 users - subfields use `.keyword` from the outset. In Kibana, you can use
+`field.keyword` to perform aggregations.
+* Existing users with custom templates - most of you won't be impacted because you use a custom template.
+* Existing users with default template - Logstash does not force you to upgrade templates if one already exists. If you
+intend to move to the new template and want to use `.keyword`, you'll have to reindex existing data. Elasticsearch's
+reindexing API can help move your data from using `.raw` subfields to `.keyword`.
```
Comment (suyograo) [inline docs/static/breaking-changes.asciidoc]: Can we link `reindexing API` to https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -3,12 +3,46 @@
...
 * <<alpha3,Logstash 5.0-alpha3>>
 * <<alpha2,Logstash 5.0-alpha2>>
 * <<alpha1,Logstash 5.0-alpha1>>
 
+[[beta1]]
+=== Logstash 5.0-beta1 Release Notes
+
+* Migrated Logstash's internal logging framework to Log4j2. This enhancement provides the following features:
+** Support for changing the <<logging>> level dynamically at runtime through REST endpoints. New APIs have been exposed
+under `_node/logging` to update log levels. Also a new endpoint `_node/logging` was added to return all existing loggers.
```
Comment (suyograo) [inline docs/static/releasenotes.asciidoc]: `_node/settings`

**Comment by suyograo** [comment]
Minor comments

File: CHANGELOG.md (hunk 1 above)
Comment (dedemorton) [inline CHANGELOG.md]: I see `"path":"/_node/settings","status":404,"error":{"message":"Not Found"}}` when I use `_node/settings`. But when I use `_node/logging` I see `{"host":"Rhodas-MacBook-Pro.local","version":"5.0.0-beta1","http_address":"127.0.0.1:9600","acknowledged":true}` 

I'm using this curl command:
`````
curl -XPUT 'localhost:9600/_node/logging
' -d '{
    "logger.logstash.outputs.elasticsearch" : "INFO"
}'
```````
Am I doing something wrong?

Comment (dedemorton) [inline CHANGELOG.md]: @suyograo ^^

Comment (suyograo) [inline CHANGELOG.md]: @dedemorton confirmed it is `_logging`. My bad.

File: docs/static/releasenotes.asciidoc (hunk 4 above)
Comment (talevy) [inline docs/static/releasenotes.asciidoc]: spoke about this, it is in fact `_node/logging`

**Comment by suyograo** [comment]
LGTM

**Pull request closed without merging**